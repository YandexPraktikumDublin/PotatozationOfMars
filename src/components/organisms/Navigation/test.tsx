import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Navigation } from './index'

describe('<Navigation />', () => {
  it('should renders correct <Navigation />', () => {
    const children = 'Test children'

    const wrapper = shallow(<Navigation>{children}</Navigation>)

    expect(wrapper.contains(children)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
