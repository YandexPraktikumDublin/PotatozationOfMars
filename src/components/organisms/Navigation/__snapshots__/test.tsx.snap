// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> should renders correct <Navigation /> 1`] = `
<nav
  className="flex items-center justify-end gap-x-4"
>
  Test children
</nav>
`;
